### DISCUSSION TITLE
Letting bot stay in channel for a few secs

### DISCUSSION THREAD

**Discussion by rorobig** [TannerGabriel/discord-bot#152] [Ideas]
Hi

I was wondering if there's a way to let the bot stay for lets say 3 seconds after the music queue is finished?
That way it doesn't leave the channel immediately after a song is done playing.

This is more of a "quality of life" improvement

**Comment by TannerGabriel** [comment]
@rorobig Thank you for the idea! I also consider this as an improvement of the current behavior. I'm not entirely sure if this is possible using the `discord-player` library. We probably would have to add a delay parameter on the line [here](https://github.com/Androz2091/discord-player/blob/63bfb17e4bc1f846a317a49a32521b2de05cfe39/src/Structures/Queue.ts#L196).

If I find the time, I will add a PR for this.

**Reply by rorobig** [reply]
alright thanks!
